helm-charts/yunikorn-all/charts/yunikorn-k8s-shim/templates/ingress.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
have you tested if this is working?
If not, I suggest to remove it? I don't think we need this for now since the default flag is also false.